### PR TITLE
DOCSP-41125 Stable API

### PR DESCRIPTION
**Pull request by lindseymoore**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kotlin-sync/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-41125>
Staging - <https://preview-mongodblindseymoore.gatsbyjs.io/kotlin-sync/DOCSP-41125-stable-api/connect/stable-api/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: snooty.toml [hunk 1]
```diff
@@ -28,5 +29,5 @@ mdb-server = "MongoDB Server"
 stable-api = "Stable API"
 api = "https://mongodb.github.io/mongo-java-driver/{+version-number+}/apidocs/mongodb-driver-kotlin-sync"
 java-api = "https://mongodb.github.io/mongo-java-driver/{+version-number+}"
-core-api = "{+java-api+}/apidocs/mongodb-driver-core/"
 kotlin-docs = "https://kotlinlang.org"
+core-api = "https://mongodb.github.io/mongo-java-driver/{+version-number+}/apidocs/mongodb-driver-core/com/mongodb"
```
Comment (mcmorisi) [inline snooty.toml:33]: This change will break existing links that appear in the Kotlin Sync repo. You should use the following value for `core-api` and update any links you have to make sure they work. You can probably just duplicate the [`snooty.toml` file from the current main branch of the repo](https://github.com/mongodb/docs-kotlin-sync/blob/master/snooty.toml)

```suggestion
core-api = "{+java-api+}/apidocs/mongodb-driver-core/"
```

**Review by mcmorisi — CHANGES_REQUESTED** [review]
One last little thing, sending it back for another pass just to verify that links still work after you make the change, but the copy looks good to me!

File: snooty.toml (hunk 1 above)
Comment (lindseymoore) [reply snooty.toml]: Ahh thanks for catching this! Updated

**Review by rozza — APPROVED** [review]
LGTM!

**Review by mcmorisi — APPROVED** [review]
LGTM, nice work!

**Pull request merged**